crates/story/examples/fixtures/test.lua

```
@@ -0,0 +1,130 @@
+    end
+
+    return results
+end
+
+function HelloWorld:configure(newConfig)
+    for k, v in pairs(newConfig) do
+        self.config[k] = v
+    end
+end
+
+function HelloWorld:processNames(names)
```

Luacheck (1.2.0)

[warning] unused argument 'self'

(W212)